### PR TITLE
Gui add multipath descriptor

### PR DESCRIPTION
**Pull request by edouardparis**

### CODE REVIEW

File: gui/Cargo.toml [hunk 1]
```diff
@@ -15,7 +15,7 @@ path = "src/main.rs"
 
 [dependencies]
 async-hwi = "0.0.4"
-liana = { git = "https://github.com/wizardsardine/liana", branch = "master", default-features = false }
+liana = { git = "https://github.com/darosior/liana", branch = "multipath_descriptors", default-features = false }
```
Comment (darosior) [inline gui/Cargo.toml:18]: It's now in master :tada:

**Comment by edouardparis** [comment]
Self-ACK ce23bcf498d2284ed389e7b229e63aeaa0c96e91

File: gui/src/app/view/coins.rs [hunk 2]
```diff
@@ -138,16 +138,16 @@ fn coin_list_view(
                                 .spacing(5)
                                 .push_maybe(if coin.spend_info.is_none() {
                                     if let Some(b) = coin.block_height {
-                                        if blockheight > b as u32 + timelock {
+                                        if blockheight > b as u32 + timelock as u32 {
                                             Some(Container::new(
-                                                text("The recovery path is available")
+                                                text("One of the recovery path is available")
```
Comment (darosior) [inline gui/src/app/view/coins.rs:143]: nit: missing an 's' to 'path'.

File: gui/src/app/view/coins.rs [hunk 3]
```diff
@@ -138,16 +138,16 @@ fn coin_list_view(
...
                                                     .bold()
                                                     .small()
                                                     .style(color::legacy::ALERT),
                                             ))
                                         } else {
                                             Some(Container::new(
-                                                text(format!("The recovery path will be available in {} blocks", b as u32 + timelock - blockheight))
+                                                text(format!("One of the recovery path will be available in {} blocks", b as u32 + timelock as u32 - blockheight))
```
Comment (darosior) [inline gui/src/app/view/coins.rs:150]: nit: missing an 's' to 'path'.

**Pull request merged**